runelite-client/src/main/java/net/runelite/client/plugins/mousehighlight/MouseHighlightOverlay.java

```diff
@@ -136,4 +136,10 @@ public Dimension render(Graphics2D graphics)
 		tooltipManager.addFront(new Tooltip(option + (Strings.isNullOrEmpty(target) ? "" : " " + target)));
 		return null;
 	}
+
+	private boolean isMenuActionRightClickOnly(int type)
```

Comment:
This should never return true for RUNELITE_OVERLAY even if the option is enabled

Reply:
Updated in commit 9997a03d76d39087fa104cabfc6504880691136d